axelrod/strategies/punisher.py

```diff
@@ -143,3 +143,42 @@ def strategy(self, opponent: Player) -> Action:
             return D
         else:
             return C
+
+
+class TrickyLevelPunisher(Player):
+    """
+    A player starts by cooperating however, after 10 rounds
+    will defect if at any point the number of defections
+    by an opponent is greater than 20%.
+
+    Names:
+
+    - Level Punisher: [Eckhart2015]_
```

Comment:
`Tricky Level Punisher` (I have not checked the reference yet but I assume that it's called that there.)